File: VB/testSample/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraBars
Imports DevExpress.XtraBars.Styles
Imports DevExpress.XtraBars.Controls
Imports DevExpress.XtraEditors
Imports DevExpress.Utils
Imports DevExpress.XtraBars.ViewInfo

Namespace testSample
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
			AddHandler tableBarItem1.TableClick, AddressOf tableBarItem1_TableClick
		End Sub
		Private Sub tableBarItem1_TableClick(ByVal sender As Object, ByVal e As CustomBarItem.TableEventArgs)
			MessageBox.Show("Size: " & e.SelectedSize)
		End Sub


	End Class
End Namespace

File: CS/testSample/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraBars;
using DevExpress.XtraBars.Styles;
using DevExpress.XtraBars.Controls;
using DevExpress.XtraEditors;
using DevExpress.Utils;
using DevExpress.XtraBars.ViewInfo;

namespace testSample
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            tableBarItem1.TableClick += new TableBarItem.OnTableClickEventHandler(tableBarItem1_TableClick);
        }
        void tableBarItem1_TableClick(object sender, CustomBarItem.TableEventArgs e)
        {
            MessageBox.Show("Size: " + e.SelectedSize);           
        }

   
    }
}
